docs/makefile-settings.md

```diff
@@ -0,0 +1,30 @@
+# Configure Makefile Tools settings
+
+Makefile Tools supports a variety of settings that can be set at the user, or workspace, level via VSCode's `settings.json` file. This topic covers the available options and how they are used.
```

Comment:
I would say `tasks.json` and `launch.json` instead of `settings.json`

Reply:
Really, we just need the Command Substitution section and an explanation that this is for tasks/launch json files.  We can consider whether it makes sense to add the rest of the sections later.

Reply:
Done. Simplified to include only command expansion but I also mentioned that variable substitution will be implemented at a later time, so users know this in advance. There were questions about that.